src/java/test/org/apache/zookeeper/server/quorum/FuzzySnapshotRelatedTest.java

```diff
@@ -60,14 +60,17 @@
 
     MainThread[] mt = null;
     ZooKeeper[] zk = null;
+    int[] clientPorts = null;
     int leaderId;
     int followerA;
 
     @Before
     public void setup() throws Exception {
+        ZooKeeperServer.setCloseSessionTxnEnabled(true);
```

Comment:
Set this to false will reproduce inconsistent pzxid bug, which is what we're going to solve here by enabling CloseSessionTxn, so we're not missing anything, just to make sure the CloseSessionTxn solved the bug here.

Reply:
This property won't impact other testcases in the class. Okay

Reply:
It would be slightly cleaner to set it back to false in @After method.
Out of interest, I find it a little bit strange to implement a feature switch to a bugfix, but on the other side I understand the concerns.

Reply:
Btw, this has already been set back to false in @after.